enemies.yaml

```
@@ -0,0 +1,76 @@
+# Generic enemies
+Goblin:
+  plural: "Goblins"
+  damage:
+    min damage: 1
+    max damage: 5
+  health:
+    min spawning health: 3
+    max spawning health: 8
+    max health level: 8
+  agility: 0.7
+  inventory:
+  - Knife
+  - Beef
+  - Chicken
+  - Raw Salmon
+  - Bone
+  - Wooden Stick
+  - Orc Iron Leggings
+  - Orc Shredded Chestplate
+  description: "The Goblin may be an easy enemy to beat, but in groups, they can be very annoying to deal with."
+Orc:
+  plural: "Orcs"
```

```suggestion
  plural: "Yrch"
```
Yes, no one calls them that. But it is the correct plural, if I spelled it correctly.